timeutil/schedule_test.go

```diff
@@ -508,6 +508,18 @@ func (ts *timeutilSuite) TestParseSchedule(c *C) {
 func (ts *timeutilSuite) TestScheduleNext(c *C) {
 	const shortForm = "2006-01-02 15:04"
 
+	// force timezone for tests to UTC otherwise if run in a
+	// different timezone where there was a daylight savings
+	// transition across one of the intervals (ie in Australia in
+	// 2019 DST started on 6th October) then the result will be
+	// different and the test will fail
+	oldLocal := time.Local
```

Comment:
Thanks for spotting this! 
(nitpick) I would tweak it a little bit and not ignore the error (just in case, I know in practice it shouldn't matter).
```go
diff --git a/timeutil/schedule_test.go b/timeutil/schedule_test.go
index 50e8e89cf3..9249301b6b 100644
--- a/timeutil/schedule_test.go
+++ b/timeutil/schedule_test.go
@@ -514,10 +514,11 @@ func (ts *timeutilSuite) TestScheduleNext(c *C) {
        // 2019 DST started on 6th October) then the result will be
        // different and the test will fail
        oldLocal := time.Local
-       local, _ := time.LoadLocation("UTC")
+       local, err := time.LoadLocation("UTC")
+       c.Assert(err, IsNil)
        time.Local = local
        defer func() {
-           time.Local = oldLocal
+               time.Local = oldLocal
        }()
 
        for _, t := range []struct {

```
(it also seems the formatting is slightly off, it seems there is a bug in our format checking, I will look at this).

Reply:
Alternatively, `testutil.Backup` can be used which makes this slightly more compact:
```go
diff --git a/timeutil/schedule_test.go b/timeutil/schedule_test.go
index 50e8e89cf3..e144bafa8d 100644
--- a/timeutil/schedule_test.go
+++ b/timeutil/schedule_test.go
@@ -26,6 +26,7 @@ import (
 
        . "gopkg.in/check.v1"
 
+       "github.com/snapcore/snapd/testutil"
        "github.com/snapcore/snapd/timeutil"
 )
 
@@ -513,12 +514,11 @@ func (ts *timeutilSuite) TestScheduleNext(c *C) {
        // transition across one of the intervals (ie in Australia in
        // 2019 DST started on 6th October) then the result will be
        // different and the test will fail
-       oldLocal := time.Local
-       local, _ := time.LoadLocation("UTC")
+       restore := testutil.Backup(&time.Local)
+       defer restore()
+       local, err := time.LoadLocation("UTC")
+       c.Assert(err, IsNil)
        time.Local = local
-       defer func() {
-           time.Local = oldLocal
-       }()
 
        for _, t := range []struct {
                schedule   string
```